.github/workflows/comment-commands.yml

```diff
@@ -0,0 +1,26 @@
+name: Comment Commands
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  pull-requests: write
```

Comment:
Why would we need PR write access?

Reply:
Because in order to re-open PRs, it must have `write` access to them